packages/calypso-products/src/features-list.tsx

```diff
@@ -1863,7 +1864,19 @@ const FEATURES_LIST: FeatureList = {
 	},
 	[ FEATURE_STATS_PAID ]: {
 		getSlug: () => FEATURE_STATS_PAID,
-		getTitle: () => i18n.translate( 'In-depth site analytics dashboard' ),
+		getTitle: () => {
+			return isEnabled( 'stats/paid-wpcom-v3' )
+				? i18n.translate( 'Stats beyond last 7 days. Detailed traffic stats and site insights.' )
+				: i18n.translate( 'In-depth site analytics dashboard.' );
+		},
+		getDescription: () =>
+			i18n.translate(
+				'Deep-dive analytics and conversion data to help you make decisions to grow your site.'
+			),
+	},
+	[ FEATURE_STATS_COMMERCIAL ]: {
+		getSlug: () => FEATURE_STATS_COMMERCIAL,
+		getTitle: () => i18n.translate( 'In-depth site analytics dashboard.' ),
```

Comment:
```suggestion
		getTitle: () => i18n.translate( 'In-depth site analytics dashboard' ),
```

I don't know why but these titles omit the full stops.